delocate/tests/test_delocating.py

```diff
@@ -2,14 +2,15 @@
 
 from __future__ import division, print_function
 
+from contextlib import contextmanager
```

Comment:
Are you using contextmanager in this function?

Reply:
Er, oops! One moment. =)